pkg/neovim

```diff
@@ -0,0 +1,27 @@
+[mirrors]
+http://github.com/neovim/neovim/archive/b6a603fe515ee1cd554a8c4151286b26209a467b.tar.gz
+
+[vars]
+filesize=8333652
+sha512=318aaa9e2054d9bfce75d63f3a2454ae6500ffb6bb2382a55b407ce1a9ee6ccb719a865a9837dd922219b37adaa52f5ba332d9d99c08df473cc44bbd85450307
+pkgver=1
+tarball=neovim-b6a603fe515ee1cd554a8c4151286b26209a467b.tar.gz
+tardir=neovim-b6a603fe515ee1cd554a8c4151286b26209a467b
+
+[deps]
+cmake
+libtool
+
+[build]
+patch -p1 < "$K"/neovim-disable-jemalloc.patch
+
+[ -n "$CROSS_COMPILE" ] && \
+  xconfflags="--host=$($CC -dumpmachine) \
+  --with-sysroot=$butch_root_dir"
+
+# cmake bugs out if its not installing from 100% clean
```

Comment:
weird, you never even called cmake, so how should make clean be needed ?
i'm pretty sure you forgot to add the cmake call.
but when doing so, please add from the existing template

Reply:
you never explicitly call cmake, its called from make

As for the make clean, I don't know. If you abort the compile mid-way through and try to resume, it would then fail the subsequent time when I was testing.